index.html

```diff
@@ -110,16 +114,14 @@ <h2>
       <!-- bottom banner  -->
       <section class="contents">
         <div class="content">
-          <h2>
-            믿을 수 있는 <br />
-            판다마켓 중고거래
-          </h2>
+          <div class="contentText">
+            <h2 class="contentTitle" style="text-align: center">
+              <span>믿을 수 있는 </span><br />
```

Comment:
💊 제안
지금 디자인에서 괜찮을 것 같지만 공백은 의도치 않은 동작을 발생시킬 수 있습니다~ 
필요한 공백이 아니면 가능하면 습관적처럼 없애는 것을 추천드려요~

```suggestion
              <span>믿을 수 있는</span><br />
```